### DISCUSSION TITLE
Zustand / persist: jsx wird falsch gerendert

### DISCUSSION THREAD

**Discussion by quargh** [neuefische/questions#106] [Web]
Hallo,

ich speichere ein boolean `isNightMode` in Zustand. Das klappt auch wunderbar. Nur wenn ich persist einsetze, wird der Zustand zwar korrekt gelesen bzw mit console gelogged, aber im jsx schlichtweg nicht umgesetzt, wenn ich einen Reload der page mache:
`<h1 className={isNightMode ? "AppTitle Night" : "AppTitle Day"}>X-Navigator</h1>` 

Liegt das evtl an next.js bzw gibt es dafür einen Workaround?

Danke und Gruß

**Comment by alexclaes** [comment]
Schickst du bitte den Link zu deinem Branch mit dem Code auf Github

**Comment by quargh** [comment]
Hier der Link: https://github.com/quargh/capstone-project/tree/Umbau

**Comment by mpagels** [comment]
Ich vermute du nutzt `useThemeStore` für den Dark/Light Mode? Dort fehlt `persist`. Das heißt, die Info wird nicht in den localStorage gechrieben. Magst du mal bitte schauen.
